docs/tutorial_bracket_strategy.md

```diff
@@ -0,0 +1,224 @@
+They are called brackets, as their orders are opening and closing like brackets:
+One order is selling `token_1` for `token_2` for a certain price, while the second order is buying `token_1` for `token_2` at a little bit lower rate.
+If the price moves in way such that both orders are traded one after the other, the passive liquidity provider earns the spread of their orders.
+
+This concept is well explained in the documentation of [the programmable-orders](https://docs.google.com/document/d/1_zTkuhWioTbRsbwKnBi5gEi3m7-bFcPyikQid7EUA2M/edit). Please study this article carefully before proceeding with this tutorial.
+
+This tutorial will cover the following steps:
+
+1. Prerequisites
+2. Setting up a MASTER-SAFE
+3. Liquidity provision
+4. Withdrawing liquidity
+5. Even more secure setup with more owners
+
+If you have any questions, don't hesitate to visit us in the [Gnosis discord channel](https://chat.gnosis.io/)
+
+## Prerequisites
+
+In this tutorial uses mainly two tools: The normal truffle scripts to build complicated and bundled ethereum transactions and the Gnosis-Safe interface to sign most of these transactions. In the Gnosis-Safe interface, we will generate a MASTER_SAFE, which will own all other accounts required for the liquidity provision and has full control over the funds involved in the liquidity provision.
+
+The prerequisites to this tutorial are:
+
+1.  Have a _proposer-account_: You should generate a new Ethereum-account via your preferred tool (e.g., MetaMask) and fund this account with a little bit of Ether. 0.2 ETH should suffice.
+    This account will be used for two tasks: Firstly, it will be used to deploy new proxy-contracts. Secondly, it is used to propose transaction to the Gnosis-Safe interface.
+
+2.  Have a _signer-account_: You should have one designated signer Ethereum-account. This account can be an already given account, and most people would probably prefer to use their usual MetaMask account. This signer-account will be added as an owner to your MASTER_SAFE and you will sign transactions with it.
+
+3.  Investment-tokens: Have the tokens you want to invest in the liquidity provision ready.
+
+4.  Token-listing: Once you have decided for a token pair to provide liquidity for, you need to make sure that both tokens are already listed on the exchange. Listing is a permissionless process and is explained [here](https://docs.gnosis.io/protocol/docs/addtoken1/).
+
+## Setting up a MASTER_SAFE
+
+In the upcoming sections, all scripts are executed on Rinkeby only.
+In order to perform the steps on Mainnet, you need to do some obvious modifications. For example, you would want to go to [https://gnosis-safe.io/](https://gnosis-safe.io/) instead of [https://rinkeby.gnosis-safe.io/](https://rinkeby.gnosis-safe.io/).
```

Comment:
I think that in the documentation of something new nothing should be "obvious".
```suggestion
In order to perform the steps on Mainnet, you need to do some simple modifications. For example, you would want to go to [https://gnosis-safe.io/](https://gnosis-safe.io/) instead of [https://rinkeby.gnosis-safe.io/](https://rinkeby.gnosis-safe.io/).
```